Parser/Client/Browser.php

```diff
@@ -211,6 +212,7 @@ class Browser extends ClientParserAbstract
         'Opera'              => array('OP', 'OM', 'OI', 'ON', 'OO'),
         'Safari'             => array('SF', 'MF'),
         'Sailfish Browser'   => array('SA'),
+        'Tenta Browser'      => array('TB')
```

Comment:
I'm not sure if this is a separate browser family as I assume this is based on the chromium android source, right?

Reply:
Please check our [github account](https://github.com/tenta-browser) that you will find most of the engine and security components are opensourced. We are opensourcing every component that is ready to be public. So, not for all parts but the goal is to be fully opensourced. Also about being a browser family, we are beasd on Chromium with our own encrypted communication layer. So, yeah I didn't understand the family part at first look. Will change it and push again. Thanks for feedback @Findus23